src/plugins/tree/treeBodyChildren.tmpl.html

```diff
@@ -7,7 +7,7 @@
             <a ng-disabled="!childrenRows || childrenRows.length === 0" data-nodrag
                class="gantt-tree-handle-button btn btn-xs"
                ng-class="{'gantt-tree-collapsed': collapsed, 'gantt-tree-expanded': !collapsed}"
-               ng-click="toggle()"><span
+               ng-click="toggleCollapse()"><span
```

Comment:
Instead of creating a new `toggleCollapse` function, use `ng-click="!isCollapsedDisabled() || toggle()"`. This will ensure that `ng-click` and `ng-disabled` are consistent.